### ISSUE TITLE
WMWE and version list - should see new legal record and context

### ISSUE THREAD

**Issue by micheal-w-wells** [bcgov/range-web#800]
![IMG_7991.jpeg](https://images.zenhubusercontent.com/5d937161c8e4d80001227881/0e153136-d9d7-4ddd-ab76-9d1f41025900)

_Originally posted by @LisaMoore1 in https://github.com/bcgov/range-web/issues/796#issuecomment-628303696_

What we want to see is another legal version that looks like last approved one, with the Reason column indicating that the last amendment hit WMWE

**Comment by LisaMoore1** [comment]
Here are the “funnies” I found with versions today. Notes are on the images. 

Plan date not matching version end date
![IMG_8003.jpeg](https://images.zenhubusercontent.com/5d937161c8e4d80001227881/81a5b3e6-b464-470e-8f15-7dc73c234eb9)

Not clear when a “stands - not reviewed” version becomes a “stands-reviewed” 
![IMG_8015.jpeg](https://images.zenhubusercontent.com/5d937161c8e4d80001227881/c3146f00-9030-474c-9cca-cf1c3a04c9df)

![IMG_8020.jpeg](https://images.zenhubusercontent.com/5d937161c8e4d80001227881/f5fa3d0c-0002-4ed0-a022-9940767db974)

**Comment by micheal-w-wells** [comment]
Date:  fixed in the back end and i botched the front end fix, sorry about that.  Getting addressed in another issue.  My eyes showed me what I wanted to see there.

Stands & the legal version list:  We don't want to show a bunch of stands records here because we only care about the dates of the first one, so that's why we see the record that we do.

What about 'Not Reviewed' not changing?  The short of it is, we could change it to just say 'Stands' very easily.  The SQL needed for that particular legal version to have its 'Stands' status update through the workflow and have it all work for multiple amendments is non-trivial.

**Comment by micheal-w-wells** [comment]
The other piece you noted after doing WMWE will get addressed as part of this issue (see original comment at the top).

**Comment by LisaMoore1** [comment]
Is there something you need from me for this one Micheal? I’m still a little confused about which versions the users see but I trust that you’ve got that covered. Catch me on this one later or all good?

**Comment by micheal-w-wells** [comment]
I think we're ok, I'm going to change it to just show 'Stands' on the legal version list for now.

If I discover a good way to keep it updated with 'needs review/reviewed' etc while keeping the right date (and not confusing future devs) I will do so at that time.

**Comment by micheal-w-wells** [comment]
Note for @calebissharp and I: 

@calebissharp  will help me sort out the back end changes related to creating another legal version with the right content, & I will work to populate the legal reason column with something appropriate.

**Comment by micheal-w-wells** [comment]
@calebissharp the above note reflects what you are doing now right?  want me to make an issue for you?

**Comment by calebissharp** [comment]
@micheal-w-wells Yep, and I think this issue tracks well enough what I'm working on.

**Comment by LisaMoore1** [comment]
@micheal-w-wells I’m wondering if we should talk about this one. I think I might be confused. I’ve made notes on the images below but I’m not sure they will make sense. 

![IMG_8046.jpeg](https://images.zenhubusercontent.com/5d937161c8e4d80001227881/2faecf10-ee78-42df-9ebc-4e297063ac92)

![IMG_8047.jpeg](https://images.zenhubusercontent.com/5d937161c8e4d80001227881/7d532c70-881f-4e42-8b5f-a398be18adf3)

I think it might be sort of related to this one as well...although I think it is likely a separate issue sort of related to #822 
![IMG_8048.jpeg](https://images.zenhubusercontent.com/5d937161c8e4d80001227881/3f8b126c-2310-479f-b70d-c51b474c8fed)

**Comment by micheal-w-wells** [comment]
Because of the many many manual steps needed to do this one I'm going to wait until I've again got cypress working on my machine (dealing with 7700 currently), to advance our UI tests to help me iron this one out.  I'm also putting #633 ahead of this one based on issues we are seeing in the myrange inbox.

Apologies for leaving this one in the 'in progress' over last week - lost a few days to holiday, half day to being sick, and a few to project handover and have neglected the myrange board.

**Comment by calebissharp** [comment]
@micheal-w-wells Do you remember what was actually left with this issue?
